tests/rptest/services/producer_swarm.py

```diff
@@ -21,6 +21,10 @@ class ProducerSwarm(Service):
     EXE = "client-swarm"
     LOG_PATH = "/opt/remote/var/client-swarm.log"
 
+    # client swarm throttles producer startups to one every 33 ms by default,
+    # see client_spawn_wait_ms in client-swarm
+    DEFAULT_CLIENT_SPAWN_WAIT_MS = 33
```

Comment:
Pass this as an argument to client-swarm so that it always reflects reality?

Reply:
Yeah this occurred to me but I was being lazy here :), but yeah.

Reply:
This is fixed in https://github.com/redpanda-data/redpanda/compare/5d9ad58e47691dbc16c82a9e1b5b413cab4307e0..837694d18838e6329dab858c39c2e930e831ccfc